plugins/woocommerce/client/blocks/assets/js/blocks/reviews/reviews-by-category/index.tsx

```diff
@@ -34,6 +34,9 @@ registerBlockType( 'woocommerce/reviews-by-category', {
 		'woocommerce'
 	),
 	supports: {
+		interactivity: {
+			clientNavigation: false,
```

Comment:
I think that we can enable it.

Reply:
You're right, enabling also Reviews by Product and All Reviews